qttypes/Cargo.toml

```diff
@@ -31,6 +31,8 @@ qtmultimediawidgets = []
 qtsql = []
 # Link against QtTest
 qttest = []
+# Don't link QtWidgets and use QGuiApplication
+nowidgets = []
```

Comment:
In Rust, features need to be additive.
So this should be a `qtwidget` feature and enable it by default.

Reply:
But then how about users who already have no-default-features enabled? It would break the build for them

Reply:
Right.
I suppose we will have to increase the semver then.

Reply:
Something like https://github.com/rust-lang/rfcs/pull/3146  would be nice.